examples/Feeds/RemoveFlightsFeedItemAttributeValue.cs

```diff
@@ -0,0 +1,307 @@
+            RemoveFlightsFeedItemAttributeValue codeExample =
+                new RemoveFlightsFeedItemAttributeValue();
+            Console.WriteLine(codeExample.Description);
+
+            // The Google Ads customer ID for which the call is made.
+            long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            // The Feed ID to which the feed items belong.
+            long feedId = long.Parse("INSERT_FEED_ID_HERE");
+
+            // The ID of the feed items to be updated.
+            long feedItemId = long.Parse("INSERT_FEED_ITEM_ID_HERE");
+
+            // The flight placeholder field name for the attribute to be removed.
+            string flightPlaceholderFieldName = "INSERT_FLIGHT_PLACEHOLDER_FIELD_NAME_HERE";
+
+            codeExample.Run(new GoogleAdsClient(), customerId, feedId, feedItemId,
+                flightPlaceholderFieldName);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description => "Removes feed items from a feed.";
+
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads API client.</param>
+        /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
+        /// <param name="feedId">The Feed ID to which the feed items belong.</param>
+        /// <param name="feedItemId">The ID of the feed items to be updated.</param>
+        /// <param name="flightPlaceholderFieldName">The flight placeholder field name for the
+        ///     attribute to be removed.</param>
+        public void Run(GoogleAdsClient client, long customerId, long feedId, long feedItemId,
+            string flightPlaceholderFieldName)
+        {
+            // Creates the Feed Item service client.
+            FeedItemServiceClient feedItemServiceClient = client.GetService(
+                Services.V4.FeedItemService);
+
+            // Strip any special characters and whitespace from the flight placeholder field name.
+            Regex specialCharsAndWhitespaceRegex = new Regex("[*'\",_&#^@\\s]");
+            flightPlaceholderFieldName = specialCharsAndWhitespaceRegex.Replace
+                (flightPlaceholderFieldName, string.Empty);
+
+            try
+            {
+                // Gets a map of the placeholder values to feed attributes.
+                Dictionary<FlightPlaceholderField, FeedAttribute> placeholdersToFeedAttributesMap =
+                    GetFeed(client, customerId, ResourceNames.Feed(customerId, feedId));
+
+                // Removes the attribute from the feed item.
+                FeedItem feedItem = RemoveAttributeValueFromFeedItem(client, customerId,
+                    placeholdersToFeedAttributesMap,
+                    ResourceNames.FeedItem(customerId, feedId, feedItemId),
+                    (FlightPlaceholderField) Enum.Parse(typeof(FlightPlaceholderField),
```

Comment:
Can you extract this enum.parse into a variable of its own? It is better for reading purposes.

Reply:
Done.